src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProjectTracker.cs

```diff
@@ -695,69 +695,72 @@ private void OutputToOutputWindow(string message)
                 return null;
             }
 
-            var solution7 = (IVsSolution7)_vsSolution;
-            if (!solution7.IsDeferredProjectLoadAllowed(projectFilename))
+            var solution5 = (IVsSolution5)_vsSolution;
+
+            // If the index is stale, it might give us a path that doesn't exist anymore that the
+            // solution doesn't know about - be resilient to that case.
+            Guid projectGuid;
+            try
             {
-                return null;
+                projectGuid = solution5.GetGuidOfProjectFile(projectFilename);
+            }
+            catch (ArgumentException)
+            {
+                var message = $"Failed to get the project guid for '{projectFilename}' from the solution, using  random guid instead.";
```

Comment:
AnyCode checked and passed only valid commandLineInfo before; now we pass all so that Roslyn can reload those projects that don't have valid commandLineInfo.